### PR TITLE
Removed lib files from Jasmine Project and integrated with current solution

### PR DESCRIPTION
**Pull request by SwapnilM92**
Removed lib files from Jasmine Project and integrated with current solution

### CODE REVIEW

**Comment by WilsonReddy** [comment]
I could not find bower.json or project.json to load dependencies. Can you include the file to restore dependencies?

**Comment by SwapnilM92** [comment]
The bower.json and project.json files are present under 'Microsoft.Legal.MatterCenter.Web' project.
For Jasmine test project we don't need these configuration files to run this test project. Hence we didn't included those files.

File: tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Jasmine/JasmineTest.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <title>Jasmine</title>
+    <link rel="shortcut icon" type="image/png" href="lib/jasmine-2.4.1/jasmine_favicon.png" />
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.4.8/angular.min.js"></script>
+    <script src="http://matterwebapp.azurewebsites.net/lib/jasmine-2.4.1/jasmine.js"></script>
+    <link href="http://matterwebapp.azurewebsites.net/lib/jasmine-2.4.1/jasmine.css" rel="stylesheet" />
```
Comment (WilsonReddy) [inline tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Jasmine/JasmineTest.html]: remove absolute path https://matterwebapp.azurewebsites.net from src. The site may be deployed any and location and should developers should able to run from there local machine.

File: tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Jasmine/JasmineTest.html [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    <script src="http://matterwebapp.azurewebsites.net/lib/jasmine-2.4.1/jasmine-html.js"></script>
+    <script src="http://matterwebapp.azurewebsites.net/lib/jasmine-2.4.1/boot.js"></script>
+    <script src="http://matterwebapp.azurewebsites.net/lib/jasmine-2.4.1/console.js"></script>
```
Comment (WilsonReddy) [inline tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Jasmine/JasmineTest.html]: I don't seed jasmine-2.4.1 dependency bower.json.

**Comment by SwapnilM92** [comment]
Closing this request as we are merging all changes in single pull request.
